webapp/locales/de.json

```diff
@@ -452,6 +452,7 @@
         "filterALL": "Alle Beiträge anzeigen",
         "success": "Gespeichert!",
         "languageSelectLabel": "Sprache",
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-12-19T07:10:45Z" title="Thursday, December 19th 2019, 8:10:45 am +01:00">Dec 19, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -452,6 +452,7 @@
         "filterALL": "Alle Beiträge anzeigen",
         "success": "Gespeichert!",
         "languageSelectLabel": "Sprache",
```


In the english version it is named:
`"languageSelectLabel": "Language of your contribution“`

In german that would be:
"languageSelectLabel": „Sprache deines Beitrags“

I would like to change this, because a lot of people seem to think this is as well the language of the web app and not of the post.
But anyways. This doesn’t hold me back to approve and we can even do it in an other PR.

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-12-19T08:02:46Z" title="Thursday, December 19th 2019, 9:02:46 am +01:00">Dec 19, 2019</time>_

---


https://github.com/Human-Connection/Human-Connection/issues/2583